### ISSUE TITLE
Add post_reboot_delay option to win_updates module

### ISSUE THREAD

**Issue by abdel-redhat** [ansible-collections/ansible.windows#518]
##### SUMMARY
Adding a  "post_reboot_delay" option to the win_updates module. 

##### ISSUE TYPE
- Feature Idea

##### COMPONENT NAME

win_updates

##### ADDITIONAL INFORMATION

This will help allowing sometime for the Windows system to settled after the automatic reboots during patching and thus avoiding unexpected behavior and issues (such as winrm send_input failures)  when the reboot option is used.

```yaml

```

**Comment by agibson2** [comment]
This looks similar to the issue #516 that I created.  The problem I had with this issue was that Ansible would get stuck when it hit this problem.  It wouldn't output anything further to the console.  I waited hours and it never completed or went further after showing a WARNING error.  Adding a delay might fix it completely for all I know but if the delay value that we use isn't correct for all circumstances, Ansible could get stuck again.  I am hoping the source of the error can get trapped or something so that the problem host will fail or maybe retry it.
